src/runtime_src/core/pcie/windows/mgmt.cpp

```diff
@@ -210,7 +211,7 @@ struct mgmt
   }
 
   void
-  get_bdf_info(uint16_t bdf[3])
+  get_bdf_info(uint16_t bdf[xrt_core::query::bdf::max])
```

Comment:
Probably better to use 
```
xrt_core::query::pcie_bdf::dbdf
get_bdf_info() 
{
 ...
};
```
where dbdf is a struct.     Either use the tuple of 4 entries or switch everything to embedded struct.

Reply:
I removed enum and using indexes only.